File: java/app/src/main/java/studentdb/model/StatusData.java
package studentdb.model;

/**
 * Represents the current status of a student's attendance in a course.
 */
public enum StatusData {

    /**
     * Student is currently attending the course.
     */
    ATTENDING,

    /**
     * Student is scheduled to attend the course in the future.
     */
    SCHEDULED,

    /**
     * Student has finished the course.
     */
    FINISHED
}

File: python/studentdb/model/StatusData.py
from enum import Enum


class StatusData(Enum):
    """
    Represents the current status of a student's attendance in a course.
    """

    ATTENDING = "ATTENDING"
    """Student is currently attending the course."""

    SCHEDULED = "SCHEDULED"
    """Student is scheduled to attend the course in the future."""

    FINISHED = "FINISHED"
    """Student has finished the course."""
